doc/api/brotli.md

```diff
@@ -97,17 +98,15 @@ const http = require('http');
 const fs = require('fs');
 http.createServer((request, response) => {
   const raw = fs.createReadStream('index.html');
-  let acceptEncoding = request.headers['accept-encoding'];
-  if (!acceptEncoding) {
-    acceptEncoding = '';
-  }
+  let acceptEncoding = request.headers['accept-encoding'] || '';
```

Comment:
May be `const` now)

Reply:
If only eslint for vscode checked markdown too.

Reply:
Can [eslint-plugin-markdown](https://github.com/eslint/eslint-plugin-markdown) be of any help? We use it in CI and local linting.

Also, you can just run from the project root for linting code fragments in docs:
```
node tools/node_modules/eslint/bin/eslint.js --ext=.md doc
```